### DISCUSSION TITLE
I am not able to integrate cv-plot into my CMake project

### DISCUSSION THREAD

**Discussion by pvillacorta** [Profactor/cv-plot#29] [Q&A]
Hi, 
I am new using C++ with CMake (in fact, it is a Qt/QML project so my my intention is to integrate cv-plot in a GUI), and I have tried to add CvPlot directory into my project directory, and then include it by using this in my root CMakeLists.txt:

add_subdirectory(CvPlot)

However, I do not know what to do next. How do I include the headers in C++? As I say, I am so new to this so I would need a step by step guide... 

I am using QtCreator, and when I open CvPlot/CMakeLists.txt it appears like this:
![Screenshot_20220527_115157](https://user-images.githubusercontent.com/93908925/170676885-1366acec-3692-4a8b-8734-c5f68f822e09.png)

I have also tried to integrate cv-plot with the Header-only method, but it does not work either.

Thanks for your help,
Pablo

**Comment by wpalfi** [comment]
Hi Pablo,
use the header-only-integration as explained here: https://github.com/Profactor/cv-plot/blob/master/doc/integration.md
It should be easy to add the include directory in qt creator. (I cannot help with that.)
